pages/join/04-contact.tsx

```diff
@@ -47,7 +52,13 @@ export default function JoinStep4({ pageTitle }) {
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState<ErrorMessageProps>(undefined);
 
+  const [linkedInPicture, setLinkedInPicture] = useState<string>("");
+  const [savePicture, setSavePicture] = useState<boolean>(true);
+  const [user, authLoading, authStateError] = useAuthState(getAuth());
+  const [loginType, setLoginType] = useState<LoginTypeNameEnum>();
+
   const createMember = async () => {
+    const token = await user.getIdToken();
```

Comment:
Ah is this how we do any admin checks?

Reply:
this isn't necessarily only for admin checks, these JWT tokens are used to verify auth in general

as we talked about on discord, we'll use [custom claims](https://firebase.google.com/docs/auth/admin/custom-claims) to assign admin permissions per user